torch/_dynamo/symbolic_convert.py

```diff
@@ -3328,19 +3351,28 @@ def STORE_GLOBAL(self, inst):
 
 class InliningGeneratorInstructionTranslator(InliningInstructionTranslator):
     generated_items: List[VariableTracker]
+    # Flag wether or not the InlineGenerator should consume the entire iterator
+    consume_all_items: bool
 
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self.generated_items = []
+        self.consume_all_items = True
```

Comment:
Can you write a TODO here that we should make this False in future, so that Dynamo closely follows what CPython does.